appveyor.yml

```diff
@@ -21,4 +21,16 @@ artifacts:
       name: Windows Build
       type: File
 
+deploy:
+    - provider: GitHub
+      repository: andreypopp/esy-solve-cudf
+      auth_token:
+          secure: TODO
```

Comment:
@andreypopp - FYI - we'll need your __encrypted__ token here:
```
GitHub authentication token (auth_token) - OAuth token used for authentication against GitHub API. You can generate Personal API access token at https://github.com/settings/tokens. Minimal token scope is repo or public_repo to release on private or public repositories respectively. Be sure to encrypt your token using the Account → Encrypt data tool.
```

Once you get your Github token - you can encrypt it with this tool on AppVeyor: https://ci.appveyor.com/tools/encrypt

Thanks & let me know if you need any help!